src/lib/Room/Room.vue

```diff
@@ -311,7 +311,7 @@ export default {
 
   computed: {
     room() {
-      return this.rooms.find(room => room.roomId === this.roomId) || this.archivedRooms.find(room => room.roomId === this.roomId) || {}
+      return this.rooms.find(room => Number(room.roomId) === Number(this.roomId)) || this.archivedRooms.find(room => Number(room.roomId) === Number(this.roomId)) || {}
```

Comment:
Pra mim é sempre um mistério se o `roomId` é `integer` ou `string`, para garantir eu to convertendo.